tools/test_configs/ESP8266Interface.json

```diff
@@ -1,5 +1,41 @@
 {
     "config": {
+        "wifi-secure-ssid": {
+            "help": "WiFi SSID for WPA2 secured network",
+            "value": "\"WIFI_SSID_SECURED\""
+        },
+        "wifi-unsecure-ssid": {
+            "help": "WiFi SSID for unsecure netwrok",
+            "value": "\"WIFI_SSID_UNSECURE\""
+        },
+        "wifi-password": {
+            "help": "WiFi Password",
+            "value": "\"WIFI_PASSWORD\""
+        },
+        "wifi-secure-protocol": {
+            "help": "WiFi security protocol, valid values are WEP, WPA, WPA2, WPA_WPA2",
+            "value": "\"WPA2\""
+        },
+        "wifi-ch-secure": {
+            "help": "Channel number of secure SSID",
+            "value": 1
```

Comment:
I guess the channel numbers should be set to 'null' so the people realize to update those

Reply:
Good point - I didn't know what to put there to keep the config compiling but force users to set something meaningful. 
'null' does the job.